qemu/tests/rdtsc_sync_test.py

```diff
@@ -0,0 +1,82 @@
+import os
+import re
+import time
+
+from avocado.utils import process
+
+from virttest import utils_misc
+from virttest import data_dir
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test rdtsc sync
+    1) Boot qemu with test binary bz1975840.flat
+    2) get a array of rdtsc value A[m]
+    3) execute "system_reset"
+    4) get a new array of rdtsc value B[n]
+    5) expected results: A[m-1] > B[0]
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    source_file = params["source_file"]
+    log_file = params["log_file"]
+    src_test_binary = os.path.join(data_dir.get_deps_dir(), source_file)
+    test_cmd = params["test_cmd"] % src_test_binary
+
+    qemu_bin = utils_misc.get_qemu_binary(params)
+    qemu_cmd = '%s %s' % (qemu_bin, test_cmd)
+    test.log.info("Send host command: %s" % qemu_cmd)
+    process.run(cmd=qemu_cmd, verbose=True, ignore_status=True, shell=True)
+    qemu_pid = process.getoutput("pgrep -f %s" % src_test_binary, shell=True)
+    if not qemu_pid:
+        test.fail("QEMU start failed!")
+
+    time.sleep(5)
+    process.run('echo -e \'{"execute":"qmp_capabilities"}'
+                '{"execute":"system_reset"}\'|nc -U /tmp/mm',
+                shell=True, verbose=True)
+    time.sleep(5)
+    process.run('echo -e \'{"execute":"qmp_capabilities"}'
+                '{"execute":"quit"}\'|nc -U /tmp/mm',
+                shell=True, verbose=True)
+
+    qemu_pid = process.getoutput("pgrep -f %s" % src_test_binary, shell=True)
+    if not qemu_pid:
```

Comment:
Seems it's a typo. The qemu process is quit, so here is qemu_pid still existed, it should fail.

Reply:
Also, you can use the exit status, when no qemu process, `pgrep -f` returns `1`.

Reply:
> Also, you can use the exit status, when no qemu process, `pgrep -f` returns `1`.

Thanks and updated, and I will keep current style to be same as above.